packages/lit-html/package.json

```diff
@@ -344,6 +346,23 @@
         "directives/*.d.ts{,.map}"
       ]
     },
+    "build:compiled:ts:tests": {
+      "#comment": "Compile the test directory with the compiler",
```

Comment:
"the compiler" -> "the lit template compiler" as we've got other tools here that compile like rollup, typescript, etc

fun fact, "compiler" has its roots in the Latin word compilo, a portmanteau of "com" (with, together) and pilo (ram down) that has connotations of both piling things up and of plundering and carrying things off

Reply:
Done! We are plundering `lit-html`'s prepare phase :P